packages/babel-core/test/config-chain.js

```diff
@@ -68,14 +68,41 @@ function escapeRegExp(string) {
   return string.replace(/[|\\{}()[\]^$+*?.]/g, "\\$&");
 }
 
+const tempDirs = [];
+
 async function getTemp(name) {
-  const cwd = await pfs.mkdtemp(os.tmpdir() + path.sep + name);
+  const tempDir = os.tmpdir() + path.sep + name;
+  tempDirs.push(tempDir);
+  const cwd = await pfs.mkdtemp(tempDir);
   const tmp = name => path.join(cwd, name);
   const config = name =>
     pfs.copyFile(fixture("config-files-templates", name), tmp(name));
   return { cwd, tmp, config };
 }
 
+afterAll(() => {
+  function deleteDir(path) {
+    // Copy from packages\babel-cli\src\babel\util.ts
+    if (fs.existsSync(path)) {
```

Comment:
We can add `rimraf` as a devDependency of `babel-core` (we already use it in other places), and just do `rimraf.sync(path)`. When we'll drop support for Node.js < 14 we can then do `fs.rmSync(path)`.